terraform/aws/components/gp2gp/gpc-consumer-locals.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  gpc-consumer_default_tags = {
+    Project = var.project
+    Environment = var.environment
+    Component = "gpc-consumer" #var.component
```

Comment:
component in tags should be the same as the var.componet and the component applied